datastore/schedule-export/src/main/java/com/example/datastore/CloudDatastoreExport.java

```diff
@@ -0,0 +1,117 @@
+package com.google.example.datastore;
+
+import com.google.appengine.api.appidentity.AppIdentityService;
+import com.google.appengine.api.appidentity.AppIdentityServiceFactory;
+import com.google.apphosting.api.ApiProxy;
+import com.google.common.io.CharStreams;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.logging.Logger;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+@WebServlet(name = "CloudDatastoreExport", value = "/cloud-datastore-export")
+public class CloudDatastoreExport extends HttpServlet {
```

Comment:
This class should probably have a more descriptive name to indicate what it is - maybe `ExportServlet` or `DatastoreExportServlet`?

Reply:
Renamed.